### ISSUE TITLE
leanpkg warning using 3.5

### ISSUE THREAD

**Issue by robertylewis** [leanprover-community/lean#105]
To configure mathlib to use 3.5.0, we set in the toml file `lean_version = "leanprover-community/lean:3.5.0"`. This lets leanpkg find the right version, but it complains:

```quote
WARNING: Lean version mismatch: installed version is 3.5.0, but package requires leanprover-community/lean:3.5.0
```

A few possible solutions:
* We can change https://github.com/leanprover-community/lean/blob/master/leanpkg/leanpkg/main.lean#L17 to split on `:` and only compare the rhs
* We maybe could change the version name to `leanprover-community/lean:3.5.0`? I'm not sure this would work.

### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already [filed](https://github.com/leanprover-community/lean/issues).
        * Specifically, check out the [wishlist](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3AI-wishlist), open [RFCs](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3ARFC),
        or [feature requests](https://github.com/leanprover-community/lean/issues?q=is%3Aissue+is%3Aopen+label%3AFeature).
    * Reduced the issue to a self-contained, reproducible test case.

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions

You can get this information from copy and pasting the output of `lean --version`,
please include the OS and what version of the OS you're running.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by cipher1024** [comment]
I don't think the second option will work but the first one should work. And because we're only talking about changing a warning, let's make sure that the warning is the only thing we touch (changing the name of the release has a lot more ramifications)

**Comment by robertylewis** [comment]
Indeed, I suspect the first option is the better choice. See #106 from yesterday. It only changes the condition to show the error message.

**Comment by robertylewis** [comment]
Closed by #106
